File: openrest4net-api/com/googlecode/openrest/WeeklyTimeWindow.cs
using System;

namespace com.googlecode.openrest
{
    class WeeklyTimeWindow
    {
        public WeeklyTimeWindow(int minuteOfWeek, int durationMins)
        {
            this.minuteOfWeek = minuteOfWeek;
            this.durationMins = durationMins;
        }

        /** Empty constructor required for initialization from JSON-encoded string. */
        internal WeeklyTimeWindow() { }

        public int? minuteOfWeek;
        public int? durationMins;
    }
}

File: trunk/openrest4j/openrest4j-api/src/com/googlecode/openrest/v1_1/WeeklyTimeWindow.java
package com.googlecode.openrest.v1_1;

import java.io.Serializable;

import org.codehaus.jackson.annotate.JsonIgnoreProperties;
import org.codehaus.jackson.map.annotate.JsonSerialize;

@JsonIgnoreProperties(ignoreUnknown = true)
public class WeeklyTimeWindow implements Serializable {
    /** Default constructor for JSON deserialization. */
    public WeeklyTimeWindow() {}

    @JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
    public Integer minuteOfWeek;
    
    @JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
    public Integer durationMins;
    
    private static final long serialVersionUID = 1L;
}
